### PR TITLE
CI: Use the `yggdrasil.x86_64` rootfs image

### PR DESCRIPTION
**Pull request by DilumAluthge**

### CODE REVIEW

**Comment by vchuravy** [comment]
We don't need it in the pipeline, but the build step defined in utils.jl will open a PR with that.

Thanks!

File: .buildkite/utils.jl [hunk 1]
```diff
@@ -81,14 +83,20 @@ end
...
-    apt install -y xz-utils
     # Don't share secrets with build_tarballs.jl
     BUILDKITE_PLUGIN_CRYPTIC_BASE64_SIGNED_JOB_ID_SECRET="" AWS_SECRET_ACCESS_KEY="" .buildkite/build.sh
     """
 
     build_plugins = plugins()
     push!(build_plugins,
+        "staticfloat/sandbox#v1" => Dict(
+            "rootfs_url" => "https://github.com/JuliaCI/rootfs-images/releases/download/v5.52/yggdrasil.x86_64.tar.gz",
+            "rootfs_treehash" => "0000000000000000000000000000000000000000", # TODO: put in the real hash
```
Comment (DilumAluthge) [inline .buildkite/utils.jl:94]: You'll need to put the actual treehash in.

Comment (vchuravy) [reply .buildkite/utils.jl:94]: Where do I find that?

Comment (DilumAluthge) [reply .buildkite/utils.jl]: Finally got around to doing this.

**Pull request merged**